django_mongodb/features.py

```diff
@@ -467,16 +326,32 @@ def django_test_expected_failures(self):
             "aggregation.tests.AggregateAnnotationPruningTests.test_unused_aliased_aggregate_pruned",
             "aggregation.tests.AggregateAnnotationPruningTests.test_referenced_aggregate_annotation_kept",
             "aggregation.tests.AggregateTestCase.test_count_star",
+            "aggregation.tests.AggregateAnnotationPruningTests.test_referenced_composed_subquery_requires_wrapping",
```

Comment:
I edited these tests (including test_exclude_multivalued_exists, but not test_optimizations) to remove the SQL inspection, so please remove them.

Reply:
Mmh. I update my branch and some of them are not passing. (test_optimizations didn't count)